bazel/boringssl_static.patch

```diff
@@ -1,24 +1,28 @@
 diff --git a/BUILD b/BUILD
```

Comment:
Note: This patch is regenerated because of changes in BoringSSL, there are no changes related to Zig C/C++ here.